src/styles/components/header.scss

```diff
@@ -7,6 +7,7 @@
   padding: 0 64px;
   position: sticky;
   top: 0;
+  z-index: 1;
```

Comment:
se calhar era boa cena começarmos ja a meter os zindexes no tema antes que fujam de controlo 🤔

Reply:
In https://github.com/makeorbreak-io/mob-web/pull/107/commits/534dc86041cc723bb4782aae80b9cbe683bc9391